src/v2/components/LocationGroupHeader.tsx

```
@@ -519,159 +519,194 @@ export const LocationGroupHeader = ({
+                        tabIndex={0}
+                        title={t('locationHeader.clickToCopy')}
+                      >
+                        <div className="flex items-center gap-2 w-full">
+                          {!isCopied ? (
+                            <PlayerList
+                              players={players}
+                              maxVisiblePlayers={maxVisiblePlayers}
+                            />
+                          ) : (
+                            <span className="text-green-400 flex items-center gap-2">
+                              <CheckIcon className="h-4 w-4" />
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)